docs/DevGuideFeatureImplementations/AddFeatureGuide.adoc

```diff
@@ -0,0 +1,44 @@
+=== Add feature
+==== Implementation
+
+The `add` feature uses the Model to check if the dish is already in the database.
+If the dish is already in the database, the meal will be added to the meal log only.
+If the dish the user wants to add is not in the database,
+then the dish will be added to both the meal log and the dish database so that the user
+can conveniently add it the next time.
+
+There are a few cases to take note:
+
+* If the dish is in the dish database:
```

Comment:
Would you like to use an Activity diagram to illustrate the alternative paths instead of a Sequence diagram? It seems like most of your Sequence diagram is not directly related to your description?